adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralRewardedAdRenderer.m

```diff
@@ -0,0 +1,161 @@
+    NSString *placementId = adConfiguration.credentials.settings[GADMAdapterMintegralPlacementID];
+
+    // if the unitId is nil.
+    if ([GADMAdapterMintegralUtils isEmpty:unitId]) {
+        NSError *error =
+        GADMAdapterMintegralErrorWithCodeAndDescription(GADMintegralErrorInvalidServerParameters, @"Unit ID cannot be nil.");
+        _adLoadCompletionHandler(nil, error);
+        return;
+    }
+    
+    if ([GADMAdapterMintegralUtils isEmpty:adConfiguration.bidResponse]) {
+        NSError *error =
+        GADMAdapterMintegralErrorWithCodeAndDescription(GADMintegralErrorInvalidServerParameters, @"bid token cannot be nil.");
+        _adLoadCompletionHandler(nil, error);
+        return;
+    }
+    
+    _rewardedAd = [MTGBidRewardAdManager sharedInstance];
+    [_rewardedAd loadVideoWithBidToken:_adConfiguration.bidResponse placementId:placementId unitId:unitId delegate:self];
+}
+
+
+
+#pragma mark MTGRewardAdLoadDelegate
+- (void)onAdLoadSuccess:(nullable NSString *)placementId unitId:(nullable NSString *)unitId {
+    
+}
+
+- (void)onVideoAdLoadSuccess:(nullable NSString *)placementId unitId:(nullable NSString *)unitId {
+    
+    if (_adLoadCompletionHandler) {
+        _adEventDelegate = _adLoadCompletionHandler(self, nil);
+    }
+}
+
+- (void)onVideoAdLoadFailed:(nullable NSString *)placementId unitId:(nullable NSString *)unitId error:(nonnull NSError *)error {
+    
+    if (_adLoadCompletionHandler) {
+        _adLoadCompletionHandler(nil, error);
+    }
+}
+
+#pragma mark MTGRewardAdShowDelegate
+- (void)onVideoAdShowSuccess:(nullable NSString *)placementId unitId:(nullable NSString *)unitId {
+    [_adEventDelegate willPresentFullScreenView];
+    [_adEventDelegate reportImpression];
+    [_adEventDelegate didStartVideo];
+}
+
+- (void)onVideoAdShowFailed:(nullable NSString *)placementId unitId:(nullable NSString *)unitId withError:(nonnull NSError *)error {
+    [_adEventDelegate didFailToPresentWithError:error];
+}
+
+- (void)onVideoPlayCompleted:(nullable NSString *)placementId unitId:(nullable NSString *)unitId{
+    [_adEventDelegate didEndVideo];
+}
+
+- (void)onVideoEndCardShowSuccess:(nullable NSString *)placementId unitId:(nullable NSString *)unitId {
+}
+
+- (void)onVideoAdClicked:(nullable NSString *)placementId unitId:(nullable NSString *)unitId {
+    [_adEventDelegate reportClick];
+}
+
+- (void)onVideoAdDismissed:(nullable NSString *)placementId unitId:(nullable NSString *)unitId withConverted:(BOOL)converted withRewardInfo:(nullable MTGRewardAdInfo *)rewardInfo {
+   
+    if (converted) {
+        GADAdReward * reward = [[GADAdReward alloc] initWithRewardType:rewardInfo.rewardName
+                                                          rewardAmount:[NSDecimalNumber decimalNumberWithString:[NSString stringWithFormat:@"%ld",(long)rewardInfo.rewardAmount]]];
+      [_adEventDelegate didRewardUserWithReward:reward];
```

Comment:
Update this to use the `didRewardedUser` API (see GMA SDK version 9.8.0 release note for more [here](https://developers.google.com/admob/ios/rel-notes#9.8.0)):
```
[_adEventDelegate didRewardUser];
```